src/main/java/exceptions/TaskNotExistException.java

```diff
@@ -0,0 +1,18 @@
+package exceptions;
+
+public class TaskNotExistException extends DukeException {
```

Comment:
You could consider renaming this MissingTaskException, which would be shorter and more grammatically correct, but this is more of a nitpick.

Reply:
Thank you for your valued recommendation, i will get to implementing that! Happy Chinese New Year!